src/senaite/jsonapi/v1/routes/registry.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from senaite.jsonapi import api
+from senaite.jsonapi import request as req
+from senaite.jsonapi.v1 import add_route
+
+
+@add_route("/senaite-config-registry", "senaite.jsonapi.v1.senaite-config-registry", methods=["GET"])
+@add_route("/registry/senaite-config-registry", "senaite.jsonapi.v1.registry.senaite-config-registry", methods=["GET"])
+def get(context, request, username=None):
+    """Senaite configuration registry records route
+    """
+    registry_records = api.get_registry_records_by_keyword("bika")
+    registry_records.update(api.get_registry_records_by_keyword("senaite"))
+
+    # Prepare batch
+    size = req.get_batch_size()
+    start = req.get_batch_start()
+    batch = api.make_batch(registry_records, size, start)
+
+    return {
+        "pagesize": batch.get_pagesize(),
+        "next": batch.make_next_url(),
+        "previous": batch.make_prev_url(),
+        "page": batch.get_pagenumber(),
+        "pages": batch.get_numpages(),
+        "count": batch.get_sequence_length(),
+        "items": registry_records,
+        "url": api.url_for("senaite.jsonapi.v1.senaite-config-registry"),
```

Comment:
Better use `api. get_batch`, which will return this dictionary immediately when you pass in the registry_records

Reply:
@ramonski I tried to use `get_batch` but it raised some errors when calling `make_items_for` so I left it as it is. I guess it is due to the fact that the `registry_records` are only strings or dicts of strings and it doesn't seem to be the kind of object `make_items_for` expects.